Solutions/TenableIO/Data Connectors/TenableStartVulnExportJob/__init__.py

```diff
@@ -8,7 +8,7 @@ def main(timestamp: int) -> object:
     tio = TenableIO()
     logging.info(
         f'requesting a new Vuln Export Job from Tenable')
-    job_id = tio.exports.vulns(updated_at=timestamp)
+    job_id = tio.exports.vulns(updated_at=timestamp, num_assets=100)

```

Comment:
Can we have `num_assets` default to 100, but be configurable to allow adjustment if necessary?

Reply:
@tomeara - Can we do that once the issue is resolved?  Even when we reduced the count to 100, customer is still getting the same error. Have you reached to customer using the customer support request they raised? Do you have any findings? Please let us know. Thanks.